build_tools/buildkite/cmake/android/arm64-v8a/pipeline.yml

```diff
@@ -25,97 +25,44 @@ steps:
 
   - wait
 
-  - label: "test on exynos-990 (mali-g77)"
-    skip: true
+  - label: "test on Galaxy S20 (exynos-990, mali-g77)"
     commands:
       - "buildkite-agent artifact download --step build build-artifacts.tgz ./"
       - "tar xzf build-artifacts.tgz"
       - "find build-android/ -name '*.cmake' -exec sed -i \"s!\\$IREE_DOCKER_WORKDIR/!\\$PWD/!g\" {} \\;"
       - "cd build-android/"
       - "ctest --output-on-failure"
     agents:
-      - "test-android=true"
       - "android-soc=exynos-990"
+      - "queue=test-android"
     env:
       IREE_DOCKER_WORKDIR: "/usr/src/github/iree"
     timeout_in_minutes: "15"
 
-  - label: "test on exynos-9820 (mali-g76)"
+  - label: "test on Galaxy S10 (exynos-9820, mali-g76)"
     commands:
       - "buildkite-agent artifact download --step build build-artifacts.tgz ./"
       - "tar xzf build-artifacts.tgz"
       - "find build-android/ -name '*.cmake' -exec sed -i \"s!\\$IREE_DOCKER_WORKDIR/!\\$PWD/!g\" {} \\;"
       - "cd build-android/"
       - "ctest --output-on-failure"
     agents:
-      - "test-android=true"
       - "android-soc=exynos-9820"
-    env:
-      IREE_DOCKER_WORKDIR: "/usr/src/github/iree"
-    timeout_in_minutes: "15"
-
-  - label: "test on snapdragon-835 (adreno-540)"
-    skip: true
-    commands:
-      - "buildkite-agent artifact download --step build build-artifacts.tgz ./"
-      - "tar xzf build-artifacts.tgz"
-      - "find build-android/ -name '*.cmake' -exec sed -i \"s!\\$IREE_DOCKER_WORKDIR/!\\$PWD/!g\" {} \\;"
-      - "cd build-android/"
-      - "ctest --output-on-failure"
-    agents:
-      - "test-android=true"
-      - "android-soc=snapdragon-835"
-    env:
-      IREE_DOCKER_WORKDIR: "/usr/src/github/iree"
-    timeout_in_minutes: "15"
-    soft_fail:
-      - exit_status: "*"
-
-  - label: "test on snapdragon-855 (adreno-640)"
-    # TODO(#4861): Re-enable when phone is fixed
-    skip: "Phone is borked. See https://github.com/google/iree/issues/4861"
-    commands:
-      - "buildkite-agent artifact download --step build build-artifacts.tgz ./"
-      - "tar xzf build-artifacts.tgz"
-      - "find build-android/ -name '*.cmake' -exec sed -i \"s!\\$IREE_DOCKER_WORKDIR/!\\$PWD/!g\" {} \\;"
-      - "cd build-android/"
-      - "ctest --output-on-failure"
-    agents:
       - "test-android=true"
```

Comment:
queue test-android?
```suggestion
      - "queue=test-android"
```

Reply:
This phone is managed by @hanhanW. So he needs to update the tag there first. Right now it's still using `test-android=true`. @hanhanW could you add the queue tag?